contracts/interfaces/pause/IProtocolPauser.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.23;
+
+/// @title ProtocolPauser
```

Comment:
can you rename it? ProtocolPauser => ProtocolPauseAdmin

Reply:
4ee3317